pkgs/tools/misc/file/default.nix

```diff
@@ -12,6 +12,7 @@ stdenv.mkDerivation rec {
     sha256 = "0l1bfa0icng9vdwya00ff48fhvjazi5610ylbhl35qi13d6xqfc6";
   };
 
+  depsBuildBuild = stdenv.lib.optional (stdenv.hostPlatform != stdenv.buildPlatform) file;
```

Comment:
I don't think `depsBuildBuild` has a target platform so `nativeBuildInputs`  should be fine? Can make unconditional too.

Reply:
This is needed to avoid infinite recursion, or at least that's why it's present.  Sounds like comment is needed, at least!

Reply:
Oh, duh!

Reply:
Oh oops. I meant to say `file` doesn't have a target platform. So yes it should be conditional bit it should also be in `nativeBuildInputs`.